helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -1025,6 +1026,23 @@ public void dropResource(String clusterName, String resourceName) {
     accessor.removeProperty(keyBuilder.resourceConfig(resourceName));
   }
 
+  @Override
+  public void addCloudConfig(String clusterName, CloudConfig cloudConfig) {
+    logger.info("Add CloudConfig to cluster {}.", clusterName);
```

Comment:
Done. Some of the fields ave been logged. Other fields such as ProcessorName and Sources have not been added since they might be null and logging them can cause an exception.